Cargo.toml

```
@@ -54,7 +54,7 @@ uuid = { version = "1.4.1", features = ["v4", "fast-rng", "macro-diagnostics", "
 wsl = { version = "0.1.0" }
 
 # v1.0.1
-libcst = { git = "https://github.com/Instagram/LibCST.git", rev = "9c263aa8977962a870ce2770d2aa18ee0dacb344", default-features = false }
+libcst = { version = "0.1.0" }
```

We should to keep the `default-features=false` because we don't need the pyo3 bindings

```suggestion
libcst = { version = "0.1.0", default-features = false }
```

This should also fix the build failures